easybuild/easyconfigs/g/GPAW/GPAW-0.11.0-customize.patch

```diff
@@ -0,0 +1,55 @@
+diff -ru gpaw-0.9.0.8965.orig/customize.py gpaw-0.9.0.8965/customize.py
```

Comment:
@akesandgren please add comments describing what this patch does, where it comes from (ref or author)